### ISSUE TITLE
officially deprecate export notebook?

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1418]
@mdboom @hamilton @wlach -- i think we should probably officially deprecate the "export bundle" features from the notebook. we are not really able to support them, and i don't think we will be any time soon. and at this point, they no longer seem as important to the workflow we want to create. i think we could revisit this down the road if we get an infusion of engineering help, but the archive thing seems like a distant concern right now as we focus in on other issues. what do you guys think?

**Comment by bcolloran** [comment]
also: this semi block #995

**Comment by mdboom** [comment]
What are the major blockers / difficulties to make this happen?  Is it primarily the iframe stuff?

I think while this doesn't necessarily help the "internal use at an organization" use case, there have been a lot of cool "remixing" this enabled on the internet at large -- throwing notebooks on Github Pages and CodePen, using with Glitch -- that made it feel like the "good parts" of Distributed Web 1.0.

tl;dr: I'd really like to revive this if we can, but I say that not really knowing what the current obstacles / amount of work is.

**Comment by hamilton** [comment]
my 2c: I agree w/ @mdboom that having a way of exporting the notebook broadly - notebook itself, files, etc. - feels like a must-have for this project, and it would feel like we are not really executing on the vision of Iodide without it and greatly reducing the value of iodide. I for one would like this functionality for some projects and have missed it over the last few months, to the point where I have to use an old version of iodide to get the functionality I want. Additionally, more than one person has asked me when exporting will become a thing again, so I'd take that as a vague sign that we're missing some key functionality by not having a solution (or plan) here.

That said, I think our _current_ "export bundle" situation is dated and doesn't reflect the direction of the project generally, and this issue (I'm assuming) is about deprecating this outdated functionality as it stands now. I'm all for that.

I'd be happy to deprecate this if we also have another issue suggesting what a more modern version of the export functionality looks like, even if the core team doesn't immediately put it on the priority stack. We should talk about export during the next core meeting and come to a consensus about this.

**Comment by bcolloran** [comment]
closed by @wlach in #1927 -- export is super stale at this point, and it will need a real rethink to be useful again, so that code is deprecated for now...
